openvdb_houdini/vdbopcustomize.py

```diff
@@ -0,0 +1,52 @@
+import hou
+import sys
+
+sopCategory = hou.sopNodeTypeCategory()
+
+dsoPaths = sys.argv[1:]
```

Comment:
Add a usage message to this script (and preferably name it something like "`buildopcustomize.py`") for anyone who wants to run it outside of CMake:
```
if not dsoPaths or any(help in dsoPaths for help in '-h -help --help'.split()):
    progname = os.path.basename(__file__))
    sys.stderr.write("""\
Usage: %(progname)s PATH [PATH...]
Which: outputs a Houdini customization script for OpenVDB nodes
Where: PATH is the pathname of a compiled OpenVDB Houdini plugin
...
""" % locals())
```
and add a comment header to the output script indicating how it was generated and maybe listing the OpenVDB version and any relevant build options.